src/main/resources/config/config.yaml

```diff
@@ -56,7 +56,8 @@ rateLimiter:  # rate = noOfReq per perMillis
 redis:
   endpoint: ${REDIS_URL:-localhost:6379}
   ssl: false
-  timeout: 2s
+  commandTimeout: ${REDIS_COMMAND_TIMEOUT:-250ms}
+  socketTimeout: ${REDIS_SOCKET_TIMEOUT:-500ms}
```

Comment:
As per the command timeout can you please add some confirmation of testing that shows this is sufficient for Publicapi to connect to Redis when is running as expected?

Since there are no tests for this can you also please add a snippet from the local testing you've done that demonstrates this connect timeout working as expected?

Reply:
Updated.